files/mods.json

```diff
@@ -703,7 +698,7 @@
         "creator": "Sychic and Lily",
         "discordcode": "skytils",
         "command": "/skytils",
-        "description": "Introduces some useful puzzle solvers, a reparty command, spam hider for Skyblock, and a ton more with lots to come in the future!",
+        "description": "Best used in tandem with NEU, Skytils provides bleeding-edge features and is almost always up to date with the latest dungeon and slayer solvers, as well as having Crystal Hollows Waypoints, useful utilities for Dungeons, and a whole lot more incredibly advanced and useful features!",
```

Comment:
> slayer solvers

Slayer "solvers" isn't exactly the correct term for this, though I do understand what you are trying to get at here. I think something like "is always up to date with solvers for the newest dungeon puzzles and assistive features for the latest slayer" or something along those lines ("assistive features" isn't exactly that great either)

Reply:
Yea definitely shorter